src/hooks/Guilds/guild/useProposalCalls.ts

```diff
@@ -35,7 +35,7 @@ const useProposalCalls = (guildId: string, proposalId: string) => {
 
   const totalOptionsNum = totalVotes?.length || 0;
   const callsPerOption = totalOptionsNum
-    ? toArray?.length / totalOptionsNum
+    ? toArray?.length / (totalOptionsNum - 1)

```

Comment:
I think this needs some commenting as well. Intuitively `callsPerOption` feels like it should be `toArray?.length / totalOptionsNum` and not `totalOptionsNum - 1`. The reason this works is because we are not getting any calls for our `Against` option.

Reply:
Hey! This was in draft when you review it. I made some changes because wasn't working well. Fell free to review now